students/synowiec_krzysztof/lesson_12_debugging/verbose_output.py

```diff
@@ -0,0 +1,55 @@
+import re
+import os
+import fnmatch
+import argparse
+import sys
+import logging
+logging.basicConfig(format='%(asctime)s - %(levelname)s- %(message)s')
+
+
+def main():
+    pattern_str, level = check_args(sys.argv[1:])
+    log_level = get_log_level(level)
+    logging.getLogger().setLevel(log_level)
+    logging.info('Choosen pattern {}'.format(pattern_str))
+    pattern = re.compile(r'{}'.format(pattern_str))
+    txt_file_list = []
+    for file in os.listdir(os.curdir):
+        if fnmatch.fnmatch(file, "*.txt"):
+            txt_file_list.append(file)
```

Comment:
lack of detailed logging info, e.g. 
"looking only for *.txt  files" 
list of found .txt files